lib/helpers.py

```diff
@@ -362,13 +362,50 @@ def formated_time():
     return "{0:04d}-{1:02d}-{2:02d}T{3:02d}:{4:02d}:{5:02d}Z".format(*ct)  # modified to fit the correct format
 
 
-def concat_state_log(machine):
+def log_switcher(log_level: str):
     """
-    Helper function to concatenate the state transition log and clear it.
-    :return comma separated state transition log string
+    Logging level switcher for handling different logging levels from backend.
+    :param log_level: ne logging level from backend
+    :return: translated logging level for logger
     """
-    state_log = ""
-    for lines in machine.timeStateLog:
-        state_log += lines + ","
-    machine.timeStateLog.clear()
-    return state_log.rstrip(",")
\ No newline at end of file
+    switcher = {
+        'error': logging.ERROR,
+        'warning': logging.WARNING,
+        'info': logging.INFO,
+        'debug': logging.DEBUG
+    }
+    return switcher.get(log_level, logging.INFO)
+
+
+def state_switcher(state: str):
```

Comment:
This name might be misleading/sounds like it has something to do with actually changing a state. "get_state_name" or something similar might be a better choice in this case

Reply:
translate_backend_state_name()